mobx/test/action_test.dart

```diff
@@ -327,5 +327,23 @@ void main() {
 
       d();
     });
+
+    test('action should wrap in transaction', () {
```

Comment:
Since a new test is introduced. Let's add a PR for version bump and changelog entry @amondnet